src/AbstractPathMappingRepository.php

```diff
@@ -173,6 +207,10 @@ protected function sortStore()
      */
     protected function createResource($filesystemPath, $path = null)
     {
+        if (0 === strpos($filesystemPath, 'k:')) {
```

Comment:
I don't know, that was your proposal :D . What do you think of `link:`? It would be clearer I think.